sympy/functions/special/delta_functions.py

```diff
@@ -499,8 +539,13 @@ def _eval_rewrite_as_sign(self, arg):
         sign
 
         """
+        if len(args) == 2:
+            arg, H0 = args[0], args[1]
+        else:
+            arg, H0 = args[0], None
         if arg.is_real:
-            return (sign(arg)+1)/2
+            if H0 == None or H0 == S.Half:
+                return (sign(arg)+1)/2
```

Comment:
why does `H0 == None` go to `sign`?  Is that for compatibility with existing behaviour?  Maybe we could rephrase the comment above?

````
The value of the second argument of Heaviside must be 1/2 for
for rewritting as sign to be strictly equivalent.  However, for compatibility
reasons, Heaviside with the second argument unspecified will also
rewrite as sign.
````
or "historical reasons"?